src/com/maddyhome/idea/vim/group/SearchGroup.java

```diff
@@ -526,7 +531,11 @@ else if (command.length() == 1) {
   }
 
   public int searchWord(@NotNull Editor editor, int count, boolean whole, int dir) {
-    TextRange range = SearchHelper.findWordUnderCursor(editor);
+    return searchWord(editor, editor.getCaretModel().getPrimaryCaret(), count, whole, dir);
+  }
+
+  public int searchWord(@NotNull Editor editor, @NotNull Caret caret, int count, boolean whole, int dir) {
```

Comment:
Let's delete `searchWord()` without a caret and let's delete all other unused functions without a caret.